style.css

```diff
@@ -0,0 +1,402 @@
+}
+.comment-btn {
+	width: 300px;
+	margin-top: 5px;
+}
+.down-social::after {
+	content: ".";
+	display: block;
+	height: 0;
+	clear: right;
+	visibility: hidden;
+}
+.article-bottom {
+	width: 100%;
+	padding-left: 3%;
+	display: flex;
+	max-width: 1200px;
+	margin: auto;
+}
+
+.article-bottom-container {
+	width: 70%;
+	display: flex;
+	border-top: 1px solid rgb(0, 0, 0);
+	max-width: 850px;
+	justify-content: space-between;
+	flex-flow: row wrap;
+}
+.article-bottom-sidebar {
+	width: 30%;
+	margin-left: 30px;
+	max-width: 315px;
+	border-top: 1px solid rgb(0, 0, 0);
+}
+.figure-container {
+	max-width: 31%;
+	padding-bottom: 20px;
+}
+.figure-image {
+	width: 100%;
+}
+.figure-image > img {
+	width: 100%;
+}
+.article-bottom-title {
+	width: 100%;
+	padding-top: 8px;
+	margin-bottom: 15px;
+}
+.figure-container a {
+	text-decoration: none;
+	line-height: 22px;
+	color: #121212;
+	font-weight: bold;
+}
+.bd-top {
+	border-top: 1px solid rgb(0, 0, 0);
+}
+
+.sidebar-bottom-elements > p {
+	padding-bottom: 10px;
+	padding-top: 20px;
+	border-top: 1px solid rgb(209, 206, 206);
+}
+
+footer {
+	margin: auto;
+	max-width: 1200px;
+	padding-bottom: 10px;
+	border-top: 4px solid rgb(226, 226, 226);
+}
+.footer-top {
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	margin-bottom: 15px;
+}
+.footer-logo {
+	margin-top: 10px;
+	width: 225px;
+	height: 30px;
+}
+
+.go-to {
+	margin-top: 20px;
+	color: #6288a5;
+	text-decoration: none;
+	font-size: 11px;
+}
+.footer-container {
+	max-width: 1200px;
+	display: flex;
+	justify-content: space-between;
+	flex-direction: row;
+}
+ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+}
+
+.footer-categories ul {
+	text-align: left;
+}
+.footer-categories ul li {
+	display: block;
+	width: 100%;
+	padding-top: 10px;
+	cursor: pointer;
+}
+.subscribe {
+	padding-left: 15px;
+	border-left: 1px solid rgb(226, 226, 226);
+}
+.subscribe-img-container {
+	width: 80%;
+}
+.subscribe-img-container > img {
+	width: 80%;
+	height: 300px;
+}
+
+.bottom-navigation {
+	text-align: center;
+	max-width: 1200px;
+	margin: auto;
+	padding-top: 9px;
+	border-top: 1px solid #ebebeb;
+	height: 55px;
+}
+.bottom-navigation ul {
+	display: flex;
+	justify-content: center;
+}
+
+.bottom-navigation ul li a {
+	font-size: 11px;
+	color: #999;
+	font-weight: 300;
+	display: block;
+	text-align: center;
+	padding: 15px 16px;
+	text-decoration: none;
+}
+.sep {
+	height: 70px;
+}
+.bottom-bar {
+	height: 60px;
+	width: 100%;
+	text-align: center;
+	border-top: 1px solid black;
+	position: fixed;
+	bottom: 0;
+	background-color: #fff;
+}
+.bottom-bar-txt {
+	display: inline-block;
+	padding-top: 20px;
+}
+.subscription-link {
+	display: inline;
+	position: absolute;
+	right: 10px;
+	padding-top: 20px;
+}
+.subscription-link > img {
+	height: 20px;
+	width: 20px;
+}
+.inflation-img {
+	display: block;
+}
+.inflation > img {
+	height: auto;
+	max-width: 100%;
+}
+
+.interactive-leading {
+	font-size: 14px;
```

Comment:
why do you have two same properties? The latter overrides the former. Choose one to use. Same for the line-height property.